### DISCUSSION TITLE
Looking for developers!

### DISCUSSION THREAD

**Discussion by fgatti675** [firecmsco/firecms#354] [General]
Hi everyone,

We are looking for freelance developer in order to help us with both with the consulting work we do for clients, as well as the development of the core library.

We have thought that what a better place to start searching that among the community :)

If you are interested, don't hesitate to let us know and reach us at hello@camberi.com

Thank you!

**Comment by Pavelkoen** [comment]
Let's discuss it!

**Reply by Pavelkoen** [reply]
belogupavel@gmail.com
